### DISCUSSION TITLE
ECL IDE- preference option -what to mention in server ip address.

### DISCUSSION THREAD

**Discussion by manonmanis471** [hpcc-systems/vscode-ecl#338] [General]
[issuesECL IDE.docx](https://github.com/hpcc-systems/vscode-ecl/files/10297917/issuesECL.IDE.docx)

**Comment by GordonSmith** [comment]
It is the IP address of the HPCC-Platform ESP server, so for https://play.hpccsystems.com:18010/esp/files/index.html it would be:
```
Server IP:  play.hpccsystems.com
SSL: ON
```
